src/views/ipv/page/page-update-name.njk

```diff
@@ -0,0 +1,92 @@
+{% set errorTitle = 'pages.pageUpdateName.content.formErrorMessage.errorSummaryTitleText' | translate %}
+{% set errorText = 'pages.pageUpdateName.content.formErrorMessage.errorSummaryDescriptionText' | translate %}
+{% set errorHref = "#pageUpdateNameOptionsForm" %}
+
+{% block content %}
+
+  <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">{{ 'pages.pageUpdateName.header' | translate }}</h1>
+  <p class="govuk-body">{{ 'pages.pageUpdateName.content.paragraph1' | translate }}</p>
+  <p class="govuk-body govuk-!-margin-bottom-2">{{ 'pages.pageUpdateName.content.paragraph2' | translate }}</p>
+
+  <ul class="govuk-list" role="list">
+    <li>
+      <ul class="govuk-list govuk-list--bullet ">
+        <li>{{ 'pages.pageUpdateName.content.list1' | translate }}</li>
+        <li>{{ 'pages.pageUpdateName.content.list2' | translate }}</li>
+      </ul>
+    </li>
+  </ul>
+
+  <p class="govuk-body">{{ 'pages.pyiNewDetails.content.paragraph3' | translate | safe}}</p>
+
+  {{ govukWarningText({
+    text: 'pages.pageUpdateName.content.warningText' | translate,
+    iconFallbackText: 'pages.pageUpdateName.content.warningTextFallback' | translate
+  }) }}
+
+  <h2 class="govuk-heading-m govuk-!-margin-top-7 govuk-!-margin-bottom-2">{{ 'pages.pageUpdateName.content.subHeading1' | translate }}</h2>
+  <p class="govuk-body govuk-!-margin-bottom-2">{{ 'pages.pageUpdateName.content.paragraph4' | translate }}</p>
+  <p class="govuk-body">{{ 'pages.pageUpdateName.content.paragraph5' | translate( { url: contactUsUrl } ) | safe}}</p>
+
+  <h2 class="govuk-heading-m govuk-!-margin-top-7 govuk-!-margin-bottom-2">{{ 'pages.pageUpdateName.content.subHeading2' | translate }}</h2>
+  <p class="govuk-body govuk-!-margin-bottom-2">{{ 'pages.pageUpdateName.content.paragraph6' | translate }}</p>
+  <p class="govuk-body govuk-!-margin-bottom-2">{{ 'pages.pageUpdateName.content.paragraph7' | translate }}</p>
+
+  <form class="govuk-heading-m govuk-!-margin-top-7 govuk-!-margin-bottom-2" id="pageUpdateNameOptionsForm" action="/ipv/page/{{ pageId }}" method="POST">
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+    {% set radiosConfig = {
+      idPrefix: "journey",
+      name: "journey",
+      fieldset: {
+        legend: {
+          text: 'pages.pageUpdateName.content.subHeading3' | translate,
+          classes: "govuk-fieldset__legend--m"
+        }
+      },
+      items: [
+        {
+          value: "update-name",
+          text: 'pages.pageUpdateName.content.formRadioButtons.updateDetailsButtonText' | translate,
+          hint: {text: 'pages.pageUpdateName.content.formRadioButtons.updateDetailsButtonTextHint' | translate}
+
+        },
+        {
+          value: "end",
+          text: 'pages.pageUpdateName.content.formRadioButtons.continueServiceButtonText' | translate,
+          hint: { text: 'pages.pageUpdateName.content.formRadioButtons.continueServiceButtonTextHint' | translate }
```

Comment:
I think we'll need to add the "OR" text inside the radio button options looking at the design/

[This](https://design-system.service.gov.uk/components/radios/#:~:text=Radio%20items%20with%20a,usually%20the%20word%20%E2%80%98or%E2%80%99.) should be what you are looking for 😄